src/firefly/js/util/VOAnalyzer.js

```diff
@@ -0,0 +1,337 @@
+/*
+ * License information at https://github.com/Caltech-IPAC/firefly/blob/master/License.txt
+ */
+
+import {get, has, isEmpty} from 'lodash';
+import Enum from 'enum';
+import {getColumn, getColumnIdx} from './../tables/TableUtil.js';
+import {MetaConst} from '../data/MetaConst.js';
+import {CoordinateSys} from '../visualize/CoordSys.js';
+
+const UCDCoord = new Enum(['eq', 'ecliptic', 'galactic']);
+
+//const pObs = 'obscore:';
+const ObsTapColName = 0;
+//const UCD = 1;
+//const Utype = 2;
+const mainMeta = 'meta.main';
+const posCol = {[UCDCoord.eq.key]: {ucd: ['pos.eq.ra', 'pos.eq.dec'],
+                                    coord: CoordinateSys.EQ_J2000},
+                [UCDCoord.ecliptic.key]: {ucd: ['pos.ecliptic.lon', 'pos.ecliptic.lat'],
+                                          coord: CoordinateSys.ECL_J2000},
+                [UCDCoord.galactic.key]: {ucd: ['pos.galactic.lon', 'pos.galactic.lat']},
+                                          coord: CoordinateSys.GALACTIC};
+
+
+
+const obsCorePosColumns = ['s_ra', 's_dec'];
+
+const ObsTapMandColumns = [
```

Comment:
Adding an explanation comment here with the link to the standard would be useful.

Reply:
Minor:  Although you are referencing ObsTap, we are really checking for any ObsCore compliant table model.  They happen to be the same.  I just wanted to point out that it does not have to come from a Tap service.

Reply:
ObsTapMandColumns is defined per ObsTAP.  Currently only the first element of each item is used as  it shows the names of the mandatory columns for ObsCore. I put the info for ObsTAP  here for the possible need of the future.